packages/manager/src/features/Kubernetes/KubernetesClusterDetail/APLSummaryPanel.tsx

```diff
@@ -50,7 +50,7 @@ const checkConsoleURL = async (
       if (error.response && error.response.status === 404) {
         setStatus({
           message:
-            'APL is still being deployed, please check back in a minute.',
+            'Installation still in progress, please check back in a minute.',
```

Comment:
```suggestion
            'Installation still in progress; please check back in a minute.',
```
Nit: these sentences are two independent clauses. Can we use a semicolon (or period) here to be [grammatically correct](https://owl.purdue.edu/owl/general_writing/punctuation/commas/commas_vs_semicolons.html)?